torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -653,6 +673,11 @@ def pre_process(self, input: torch.Tensor) -> torch.Tensor:
         return w_24.t()
 
 
+@dataclass(frozen=True)
+class MarlinQQQLayout(Layout):
```

Comment:
just a note: if you are creating a new tensor subclass, the layout/tensor_impl is not required if you feel it does not help in any way. unless you want to expose `MarlinQQQLayout` as a top level API for `int8_dynamic_activation_int4_weight`, as you do right now in the PR.

Reply:
got it.